### PR TITLE
adds logo to menu

### PR DESCRIPTION
**Pull request by boazsender**
Here is what this looks like: 
big:
image

small:
image

### CODE REVIEW

File: client/routes/Navigation.jsx [hunk 1]
```diff
@@ -34,9 +34,9 @@ const Navigation = () => {
                 <Icon name="content" />
                 MENU
             </Button>
-            {menuExpanded && (
```
Comment (rwaldron) [inline client/routes/Navigation.jsx]: Please revert this change.

**Comment by rwaldron** [comment]
Please fix the lint nits

File: client/routes/Nav.css [hunk 2]
```diff
@@ -9,3 +9,16 @@
         display: block;
     }
 }
+
+.ui.menu .item.logo {
```
Comment (rwaldron) [inline client/routes/Nav.css]: This should be named something like `.navigation__menu-item-logo`

File: client/routes/Navigation.jsx (hunk 1 above)
Comment (boazsender) [reply client/routes/Navigation.jsx]: reverted

File: client/routes/Nav.css [hunk 3]
```diff
@@ -9,3 +9,17 @@
         display: block;
     }
 }
+
+.ui.menu .item.navigation__menu-item-logo {
+  padding: 0px;
+}
+
```
Comment (rwaldron) [inline client/routes/Nav.css]: Beautifully named, thank you.
